### ISSUE TITLE
Inno Setup fails, even though extraction seemed to be successful

### ISSUE THREAD

**Issue by ltGuillaume** [Bioruebe/UniExtract2#206]
The original Universal Extractor, which also used InnoUnp, has no issues with these files.

Example setup files:
https://www.slsknet.org/SoulseekQt/Windows/

Log:
```
Checking permissions for path D:\Programs\Utilities\Universal Extractor\UniExtract.ini
Using global settings
Option: language = English
Option: batchqueue = .\batch.queue
Option: filescanlogfile = .\log\filescan.txt
Option: batchenabled = 0
Option: history = 0
Option: appendext = 0
Option: warnexecute = 1
Option: nostatusbox = 0
Option: hidestatusboxiffullscreen = 1
Option: openfolderafterextr = 0
Option: deletesourcefile = 0
Option: freespacecheck = 1
Option: timeout = 60
Option: keepoutputdir = 0
Option: keepopen = 0
Option: feedbackprompt = 0
Option: log = 0
Option: checkgame = 1
Option: sendstats = 0
Option: extract = 1
Option: unicodecheck = 1
Option: extractvideotrack = 1
Option: silentmode = 0
Option: storeguiposition = 0
Option: cleanup = 1
Option: statusposx = -1
Option: statusposy = -1
Option: addassocenabled = 0
Option: addassoc = 001,1,ace,arc,arj,bin,cab,cpio,cue,deb,dbx,gz,img,kgb,kge,lha,lit,lzh,lzo,pea,rpm,sit,uha,z
Option: addassocallusers = 0
Option: topmost = 0
Option: updateinterval = 30
Option: lastupdate = 2020/07/22
Option: ID = D0F85D9EFF16BC5DBCF56A8C6
Option: nightlyupdates = 1
Program directory: D:\Programs\Utilities\Universal Extractor
Finished loading preferences from file D:\Programs\Utilities\Universal Extractor\UniExtract.ini
Starting Universal Extractor 2.0.0 RC 3
Command line parameters: A:\SoulseekQt-2019-7-22-64bit.exe A:\SoulseekQt-2019-7-22-64bit
------------------------------------------------------------
File size: 16.86 MB
Output directory: A:\SoulseekQt-2019-7-22-64bit
File seems to be executable
Start file scan using Exeinfo PE
Reading file D:\Programs\Utilities\Universal Extractor\log\exeinfo.log
SoulseekQt-2019-7-22-64bit.exe - Inno Setup Module [SFX] - ver. (5.5.7) Borland Delphi - ( from Ovl  : zlb) - www.innosetup.com - *Mode : Lzma

Testing Inno Setup
Executing: C:\Windows\system32\cmd.exe /d /c ""D:\Programs\Utilities\Universal Extractor\bin\innounp.exe" "A:\SoulseekQt-2019-7-22-64bit.exe"
; Version detected: 5507
Compression used: lzma
Files: 54 ; Bytes: 63594699

Starting inno extraction
Executing: C:\Windows\system32\cmd.exe /d /c ""D:\Programs\Utilities\Universal Extractor\bin\innounp.exe" -x -m -a "A:\SoulseekQt-2019-7-22-64bit.exe" 2>&1 | "D:\Programs\Utilities\Universal Extractor\bin\mtee.exe" "D:\Programs\Utilities\Universal Extractor\log\teelog.txt"
           with options: showFlag = 6, initialShow = True, patternSearch = True, workingdir = A:\SoulseekQt-2019-7-22-64bit
Searching for plugin D:\Programs\Utilities\Universal Extractor\bin\mtee.exe
Pid: 6112
Runtitle: 0
Teelog:
; Version detected: 5507
#1 {app}\D3Dcompiler_47.dll
Reading slice A:\SoulseekQt-2019-7-22-64bit.exe
#2 {app}\libEGL.dll
#3 {app}\libgcc_s_seh-1.dll
#4 {app}\libGLESV2.dll
#5 {app}\libstdc++-6.dll
#6 {app}\libwinpthread-1.dll
#7 {app}\opengl32sw.dll
#8 {app}\Qt5Core.dll
#9 {app}\Qt5Gui.dll
#10 {app}\Qt5Multimedia.dll
#11 {app}\Qt5Network.dll
#12 {app}\Qt5Svg.dll
#13 {app}\Qt5Widgets.dll
#14 {app}\SoulseekQt.exe
#15 {app}\audio\qtaudio_windows.dll
#16 {app}\bearer\qgenericbearer.dll
#17 {app}\iconengines\qsvgicon.dll
#18 {app}\imageformats\qgif.dll
#19 {app}\imageformats\qicns.dll
#20 {app}\imageformats\qico.dll
#21 {app}\imageformats\qjpeg.dll
#22 {app}\imageformats\qsvg.dll
#23 {app}\imageformats\qtga.dll
#24 {app}\imageformats\qtiff.dll
#25 {app}\imageformats\qwbmp.dll
#26 {app}\imageformats\qwebp.dll
#27 {app}\mediaservice\dsengine.dll
#28 {app}\mediaservice\qtmedia_audioengine.dll
#29 {app}\platforms\qwindows.dll
#30 {app}\playlistformats\qtmultimedia_m3u.dll
#31 {app}\styles\qwindowsvistastyle.dll
#32 {app}\translations\qt_ar.qm
#33 {app}\translations\qt_bg.qm
#34 {app}\translations\qt_ca.qm
#35 {app}\translations\qt_cs.qm
#36 {app}\translations\qt_da.qm
#37 {app}\translations\qt_de.qm
#38 {app}\translations\qt_en.qm
#39 {app}\translations\qt_es.qm
#40 {app}\translations\qt_fi.qm
#41 {app}\translations\qt_fr.qm
#42 {app}\translations\qt_gd.qm
#43 {app}\translations\qt_he.qm
#44 {app}\translations\qt_hu.qm
#45 {app}\translations\qt_it.qm
#46 {app}\translations\qt_ja.qm
#47 {app}\translations\qt_ko.qm
#48 {app}\translations\qt_lv.qm
#49 {app}\translations\qt_pl.qm
#50 {app}\translations\qt_ru.qm
#51 {app}\translations\qt_sk.qm
#52 {app}\translations\qt_uk.qm
#53 {app}\translations\qt_zh_TW.qm
#54 embedded\WizardImage0.bmp
#55 embedded\WizardSmallImage0.bmp
#56 embedded\english.isl
#57 embedded\brazilianportuguese.isl
#58 embedded\catalan.isl
#59 embedded\corsican.isl
#60 embedded\czech.isl
#61 embedded\danish.isl
#62 embedded\dutch.isl
#63 embedded\finnish.isl
#64 embedded\french.isl
#65 embedded\german.isl
#66 embedded\greek.isl
#67 embedded\hebrew.isl
#68 embedded\hungarian.isl
#69 embedded\italian.isl
#70 embedded\japanese.isl
#71 embedded\norwegian.isl
#72 embedded\polish.isl
#73 embedded\portuguese.isl
#74 embedded\russian.isl
#75 embedded\scottishgaelic.isl
#76 embedded\serbiancyrillic.isl
#77 embedded\serbianlatin.isl
#78 embedded\slovenian.isl
#79 embedded\spanish.isl
#80 embedded\turkish.isl
#81 embedded\ukrainian.isl
#82 install_script.iss

Cleanup: Deleting A:\SoulseekQt-2019-7-22-64bit\
Moving files from A:\SoulseekQt-2019-7-22-64bit\{app} to A:\SoulseekQt-2019-7-22-64bit
Extraction finished, success: 0
Terminating - Status: failed
```

**Comment by ltGuillaume** [comment]
There may be some strange timing issue?

As soon as the extraction started, I tried to get into `SoulseekQt-2019-10-9-64bit\{app}` with Total Commander and I saw the files being extracted. After doing this, the files were properly moved from `{app}` into the parent folder `SoulseekQt-2019-10-9-64bit`. No error message was presented. All was well. This is reproducible.

If I _don't_ go in the `{app}` folder, I always get this error message (and the log as presented above).
![image](https://user-images.githubusercontent.com/4777345/88461413-8a175f00-cea3-11ea-83c8-9c6539d72510.png)
